### ISSUE TITLE
[Docs] Difference between Reference and Salt Module Index

### ISSUE THREAD

**Issue by srkunze** [saltstack/salt#27919]
What's the difference between https://docs.saltstack.com/en/latest/ref/index.html and https://docs.saltstack.com/en/latest/salt-modindex.html ?

Put it differently, when do I use which one? Couldn't they be merged into a single page?

**Comment by jfindlay** [comment]
@srkunze, you are right.  Those pages seem to want to solve similar (the same?) problems, and both may have evolved over time in parallel.  I, myself, was confused about the distinct intentions of either in comparison.

**Comment by rallytime** [comment]
@srkunze Perhaps these pages have changed since you last filed this issue, but the salt-modindex is a reference-point/list of all of the available modules within Salt. The index.html page (or "Reference") includes everything from the salt-modindex page, as well as references to other useful Salt documentation such as pages about using the Salt CLI, configuring the Salt Master or Salt Minions, how to use renderers, etc., etc.

The https://docs.saltstack.com/en/latest/ref/index.html page is more of a reference page for all of the salt docs while the https://docs.saltstack.com/en/latest/salt-modindex.html only contains the links to individual module docs as more of a quick reference to available modules.

Does that help explain the difference? Do you think we need to do something else to fix this issue up, or can we close it?

**Comment by srkunze** [comment]
@rallytime Let me have a look. :)

**Comment by srkunze** [comment]
Alright. I see your point. Well, would it hurt to add your explanation to the top of the page?

index = "Table of Contents of Salt docs"
modindex = "Available Salt Modules"

In any case, this makes more sense now. :) Thanks for your efforts.

**Comment by rallytime** [comment]
@srkunze We can definitely add a note to the `Reference` (index) page easily, but the modindex page is auto-generated and I'm not sure how to edit that currently. @jacobhammons said he'd look into it to see if it was possible.
